GOVERNANCE.md

```diff
@@ -54,3 +54,204 @@ Inspired by [ANSI’s Essential Requirements for Due Process](https://share.ansi
+
+
+
+
+
+## 9.0 Release Build Approval Guidelines
+
+This section covers scheduling of minor, development, and patch releases, and approvals for all builds and releases.
+
+## 9.1 Dev Release Scheduling and Approvals
+* Dev releases may be scheduled by the maintainers  to optimize development resources, based on the queue of approved PRs
+  * There is no particular desired/expected release frequency; releases may be cut as soon as there is an approved PR, or several PRs may be consolidated into a single release at the convenience of the maintainers and dev staff
+  * *Rationale:*  Dev builds are expected to change in functionality, and getting changes out as quickly as practical is usually desirable.
+  * Each Dev Release shall require the approval of one maintainer once all the PRs are approved, and the test cass all pass successfully.
+* Dev releases shall be reported in brief to the CRWG and the SWG
+
+## 9.2 Major Production Release Build & Release Approvals
+
+* Major production releases will be scheduled by the SWG as described above
+  * *(TODO:  insert a diagram of the promotion process)*
+* Each Major Production Release shall require the approval of two maintainers after the following are complete:
+  * The scope of the release is finalized and ratified by the SWG
+  * All approved PRs for the major release are complete
+  * The SWG reviews the final list of enhancements in the release and signs off on releasing the dev version into production
+
+## 9.3 Minor Production Release Scheduling and Approvals
+
+* Minor production releases may be scheduled by the maintainers based on the queue of approved PRs
+* Minor production releases to introduce enhancements should be combined  to minimize the number of production releases, targeting minor production releases to be issued around four weeks or so as long as there is a queue of approved PRs.  (This frequency can be increased in times of urgent need for new functionality).
+  * *Rationale:*   Minimizing the number of production releases will help with supportability, by reducing the number of releases that end users wishing to remain current need to consider, and reducing communications overhead.
+* Each Minor Production Release shall require the approval of two maintainers.
+* Minor Production releases shall be reported in brief to the CRWG and the SWG, 
+* A roadmap of anticipated minor releases shall be reported by the maintainers to the CRWG based on PRs that are in process.
+
+## 9.4  Production Patch Release Scheduling and Approvals
+
+* Patch releases to correct defects without releasing new functionality may be scheduled by the maintainers based on the presence of approves defect correction PRs, or other non-functional PRs (e.g. security remediations).
+*  Patch releases require the approval of one maintainer
+*  Patch releases shall be reported to the CRWG.
+
+ ## 9.5 Summary of Release Approval Requirements
```

Comment:
similar comment on the terms used in the table for Major Release, Minor Release etc. Do we want to make these consistent with the rest of this section where we refer to Major Production Release, Minor Production Release, Production Patch Release?

Reply:
agreed